guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -27,9 +27,9 @@ ifdef::satellite[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos \
---enable={RepoRHEL8AppStream} \
---enable={RepoRHEL8BaseOS} \
---enable={RepoRHEL8ServerSatelliteUtils}
+--enable={RepoRHEL9AppStream} \
+--enable={RepoRHEL9BaseOS} \
+--enable={RepoRHEL9ServerSatelliteUtils}
 ----
 . Enable the satellite-utils module:
```

Comment:
AFAIK there's no `satellite-utils` DNF module on EL9 so I think this step became redundant.

Reply:
@ekohl Should the step be reverted back to what it was or is it good as is?

Reply:
@ekohl Following up.

Reply:
@ekohl Following up again to see if this needs to be reverted back to what it was before or if it is good as is?

Reply:
Line 34 describes enabling a module. EL8 uses modules, EL9 doesn't. If you are documenting a procedure that is to be performed on EL9, you can drop this whole step. If you are documenting a procedure that can be performed on either EL8 or EL9, you can precede this step with something like "If you are <doing XYZ> on EL8, enable the satellite-utils module".